src/styles/mixins/_tables.scss

```diff
@@ -12,7 +12,15 @@
   .table-toolbar {
     display: flex;
     gap: var(--kui-space-50, $kui-space-50);
+    padding: var(--kui-space-20, $kui-space-20) var(--kui-space-20, $kui-space-20) var(--kui-space-0, $kui-space-0)
+      var(--kui-space-20, $kui-space-20); // prevents focus outline in toolbar elements from being cut off
```

Comment:
question: why is this on two lines? If it's wrapping to multiple, this is likely a setting on your local editor